### ISSUE TITLE
Property 'popup_cards' of undefined

### ISSUE THREAD

**Issue by msekoranja** [dwainscheeren/dwains-lovelace-dashboard#222]
Hi,

I have upgraded from 1.1.14 to 2.0.0. The theme works (and it great!).
However in the logs (at startup only) I get:
```
2020-11-17 23:48:12 ERROR (MainThread) [frontend.js.latest.202010214] http://z.z.z.z:8123/dwains_dashboard/js/dwains-dashboard.js:90:13826 Uncaught TypeError: Cannot read property 'popup_cards' of undefined
```
Am I missing something in configuration (due to upgrade)?

Thanks,
Matej

**Comment by t0bse** [comment]
same here, fresh install

**Comment by vikdb** [comment]
Same

**Comment by dwainscheeren** [comment]
Seems like a bug on some browser/devices but you can ignore it if everything else works fine.

**Comment by boomam** [comment]
Same here, even though I've actually removed Dwains, and it occurs a couple of times a day it seems.

Any solution to this?

**Comment by dwainscheeren** [comment]
Will be fixed in dd 3.0
